_config/config.yml

```diff
@@ -17,3 +17,5 @@ HTTP:
     must-revalidate: "true"
     no-transform: "true"
   vary: "Cookie, X-Forwarded-Protocol, User-Agent, Accept"
+Hierarchy:
+  node_threshold_total: 30
```

Comment:
For default values it's better to put these in private statics. YML is more for overrides and injector configs.